apps/www/components/Onboarding/Page.js

```diff
@@ -271,8 +360,8 @@ class Page extends Component {
                       return (
                         <Component
                           key={name}
+                          {...data}
                           name={name}
-                          user={user}
```

Comment:
Why spread the entire data object?
Wouldn't `user={data.me}` be sufficient considering the changes above?

Reply:
for the Postcard Component we need the postcard object